src/content/whats-new/2023/10/whats-new-10-02-nrql-lookups-api.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 'Manage NRQL Lookup Tables with the NRQL Lookups API'
+summary: 'Automate the maintenance of NRQL Lookup tables'
+releaseDate: '2023-10-02'
+learnMoreLink: 'https://docs.newrelic.com/docs/apis/lookups-service-api/lookups-service-api'
+---
+
+When the NRQL Lookups feature was added, lookup table maintenance was exposed to users via the
```

Comment:
How about "table maintenance was made available to users" or something like that.  "Exposed" feels jargony.